contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -149,20 +165,32 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, IsContract, AragonApp
         _increaseKeysOpIndex();
 
         uint256 activeOperatorsCount = getActiveNodeOperatorsCount();
-        if (_active)
+        SigningKeysStats memory _signingKeysStats = signingKeysStats;
+        if (_active) {
             ACTIVE_OPERATORS_COUNT_POSITION.setStorageUint256(activeOperatorsCount.add(1));
-        else
+            _setSigningKeysStats(
+                _signingKeysStats.totalSigningKeys.add(operators[_id].totalSigningKeys),
+                _signingKeysStats.usedSigningKeys.add(operators[_id].usedSigningKeys),
+                _signingKeysStats.stoppedSigningKeys.add(operators[_id].stoppedValidators)
+            );
+        } else {
             ACTIVE_OPERATORS_COUNT_POSITION.setStorageUint256(activeOperatorsCount.sub(1));
-
+            _setSigningKeysStats(
+                _signingKeysStats.totalSigningKeys.sub(operators[_id].totalSigningKeys),
+                _signingKeysStats.usedSigningKeys.sub(operators[_id].usedSigningKeys),
+                _signingKeysStats.stoppedSigningKeys.sub(operators[_id].stoppedValidators)
+            );
+        }
```

Comment:
Not sure that we can so easily exclude the keys of an inactive node operator from the statistics

Reply:
can u re-check, pls